databases/mysql8/Portfile

```diff
@@ -209,6 +209,7 @@ if {$subport eq $name} {
             copy ${prefix}/share/${name_mysql}/support-files/macports/my.cnf \
                 ${prefix}/etc/${name_mysql}/
         }
+        file link -symbolic ${prefix}/lib/${name_mysql}/lib mysql
```

Comment:
This changes what the port installs so its revision needs to be increased. However it seems like it should be possible to fix this problem without creating a symlink.

Reply:
The proposed fix is a WorkAround. The basic problem seems to be in my_openssl. I don't know enough about C/C++ programming to be able to solve it.

Also, I'm not that well versed in the macports process, so please forgive any errors I make